docs/Materials.md.html

```diff
@@ -1279,8 +1279,8 @@
     - **Screen**: blending is enabled. Effectively the opposite of the `multiply`, the content of the
       render target is brightened.
     - **Masked**: blending is disabled. This blending mode enables alpha masking. The alpha channel
-      of the material's output defines whether a fragment is discarded or not. See the maskThreshold
-      section for more information.
+      of the material's output defines whether a fragment is discarded or not. Additionally,
+      ALPHA_TO_COVERAGE is enabled. See the maskThreshold section for more information.
```

Comment:
This is irrelevant since we always now set alpha to 1 for fragments that pass the test. See below.